widgets/text/text.go

```diff
@@ -108,6 +109,25 @@ func (t *Text) Write(text string, wOpts ...WriteOption) error {
 	if opts.replace {
 		t.reset()
 	}
+
+	// If set, limit the buffer (if maxContent has been set)
+	if len(t.content) >= t.opts.maxContent && t.opts.maxContent > 0 {
+		fmt.Printf("len: %d strlen: %d", len(t.content), len(t.content))
```

Comment:
Was this a debug print that we want to remove before we merge this in?

Reply:
Yes, just debug which crept in.